precompile/contracts/feemanager/contract_test.go

```diff
@@ -258,3 +332,14 @@ func TestFeeManager(t *testing.T) {
 func BenchmarkFeeManager(b *testing.B) {
 	allowlist.BenchPrecompileWithAllowList(b, Module, state.NewTestStateDB, tests)
 }
+
+func getGasCost(oldFeeConfig commontype.FeeConfig, feeConfig commontype.FeeConfig) uint64 {
+	// address doesn't matter for gas cost
+	_, data, err := PackFeeConfigChangedEvent(common.Address{}, oldFeeConfig, feeConfig)
+	if err != nil {
+		panic(err)
+	}
+
+	logGasCost := GetFeeConfigGasCost + FeeConfigChangedEventBaseGasCost + contract.LogDataGas*uint64(len(data))
```

Comment:
All of the fields of the event are a fixed size, so this should be a constant value regardless of the inputs

Reply:
yep, changed them